app/controllers/awards_controller.rb

```diff
@@ -0,0 +1,47 @@
+class AwardsController < ApplicationController
+  before_action :load_award, only: [:update, :destroy]
+
+  def award
+    redirect_to admin_awards_path
```

Comment:
Kāpēc šis redirekts eksiste? Varbūt tev kaut kur jānomaina pats links?

Reply:
tagad kad palika zem viena namespace tad viss ir kartiba, pirmstam vienkarsi palika URL no ta create, un ja refreshoja lapu tad radija ka neeksiste tada [GET] /awards ...